base/compiler/inferencestate.jl

```diff
@@ -28,9 +28,7 @@ mutable struct InferenceState
     pc´´::LineNum
     nstmts::Int
     # current exception handler info
-    cur_hand #::Union{Nothing, Pair{LineNum, prev_handler}}
-    handler_at::Vector{Any}
-    n_handlers::Int
+    handler_at::Vector{LineNum}
```

Comment:
I think "handler_at" naming is actually a bit confusing ? To me "entered_at" sounds more reasonable ("handler" usually mean `catch` clause ... ?).

Reply:
I think it is the same to say it is handled by the `try` statement. The `catch` / `leave` is where we stop handling errors (which is why it is simpler to point to the `try` statement also, since we have more information there)